resources/agent.rb

```diff
@@ -49,12 +49,18 @@
     package 'sensu-go-agent' do
       if latest_version?(new_resource.version)
         action :upgrade
+        notifies :reload, 'systemd_unit[sensu-agent]', :immediately
```

Comment:
I missed the failing CI. Here is a bit more flavor on what it means and what it's asking for in terms of change. We are calling the `reload` action in the `if` and `else` conditionals so it should be moved outside of the conditional since the result is the same and is a bit more DRY.

```ruby
package sensu-go' do
  if condition1?
    do_stuff
  else
    do_other_stuff
  notifies :reload,  'systemd_unit[sensu-agent]', :immediately
end
```

Reply:
Yep that makes sense to me, change is in whenever you get time to verify.  Thanks!